server/schedule/region_scatterer.go

```diff
@@ -153,6 +179,8 @@ func (r *RegionScatterer) scatterRegion(region *core.RegionInfo) *operator.Opera
 	}
 
 	scatterWithSameEngine(ordinaryPeers, r.ordinaryEngine)
+	// FIXME: target leader only consider the ordinary engine.
+	targetLeader := r.collectAvailableLeaderStores(targetPeers, r.ordinaryEngine)
```

Comment:
Will the leader store be collected again after `collectAvailableLeaderStores` in `scatterWithSameEngine` function?

Reply:
now, only consider the dorinaryEngine.